index.html

```diff
@@ -67,50 +92,55 @@ <h2 style="margin: 0;">Enjoying my site?</h2>
                     </div>
                 </div>
             </div>
-            <div v-if="currentElement">
-                <h2 style="text-align: center;">
-                    <div class="item" style=" padding: 13px 15px 12px" @click="navigateTo('')">
-                        {{ data.index[currentElement][0] }}&nbsp;{{ data.index[currentElement][1] }}
-                    </div>
-                </h2>
+            <transition name="slide-up" mode="out-in">
+            <div key=0 v-if="currentElement">
+                <transition :name="slideTransition" mode="out-in">
+                    <h2 :key="currentElement" style="text-align: center;">
```

Comment:
Make sure clicking this takes you back to the home screen

Reply:
I could be wrong, but I don't think that is necessary anymore. Now that the search bar is persistent, that is the clear an obvious thing to click to go back into search mode, and if you want to go all the way home, clicking on the menu instead of the currently active item makes a lot more sense.

If you want, I'm happy to readd that functionality, it just seems counter intuitive that it should work like that, especially now that there are better options